crates/aptos-api-tester/src/wrappers.rs

```diff
@@ -0,0 +1,70 @@
+// Copyright © Aptos Foundation
+
+use crate::{
+    tests::{test_cointransfer, test_newaccount, test_nfttransfer, test_publishmodule},
+    utils::{
+        create_account, create_and_fund_account, get_client, get_faucet_client, NetworkName,
+        TestFailure,
+    },
+};
+use anyhow::Result;
+use aptos_sdk::{coin_client::CoinClient, token_client::TokenClient};
+
+pub async fn wrapper_newaccount(network_name: NetworkName) -> Result<(), TestFailure> {
+    // spin up clients
+    let client = get_client(network_name);
+    let faucet_client = get_faucet_client(network_name);
+
+    // create and fund account
+    let account = create_and_fund_account(&faucet_client).await?;
+
+    // run test
+    test_newaccount(&client, &account, 100_000_000).await
+}
+
+pub async fn wrapper_cointransfer(network_name: NetworkName) -> Result<(), TestFailure> {
```

Comment:
Wrapper might be better as a different word, since this is the test setup and all, but I don't have a good suggestion right now